sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationClient.cs

```diff
@@ -0,0 +1,1751 @@
+        /// </param>
+        /// <param name="context"> Request context. </param>
+        internal virtual Response GetLanguages(string clientTraceId = null, string scope = null, string acceptLanguage = null, string ifNoneMatch = null, RequestContext context = default)
+        {
+            ETag? eTag = null;
+            if (ifNoneMatch != null)
+            {
+                eTag = new ETag(ifNoneMatch);
+            }
+
+            return this.GetLanguages(clientTraceId, scope, acceptLanguage, eTag, context);
+        }
+
+        // Overrides for generated class
+
+        /// <summary> Gets the set of languages currently supported by other operations of the Translator. </summary>
+        /// <param name="clientTraceId"> A client-generated GUID to uniquely identify the request. </param>
+        /// <param name="scope">
+        /// A comma-separated list of names defining the group of languages to return.
+        /// Allowed group names are: `translation`, `transliteration` and `dictionary`.
+        /// If no scope is given, then all groups are returned, which is equivalent to passing
+        /// `scope=translation,transliteration,dictionary`. To decide which set of supported languages
+        /// is appropriate for your scenario, see the description of the [response object](#response-body).
+        /// </param>
+        /// <param name="acceptLanguage">
+        /// The language to use for user interface strings. Some of the fields in the response are names of languages or
+        /// names of regions. Use this parameter to define the language in which these names are returned.
+        /// The language is specified by providing a well-formed BCP 47 language tag. For instance, use the value `fr`
+        /// to request names in French or use the value `zh-Hant` to request names in Chinese Traditional.
+        /// Names are provided in the English language when a target language is not specified or when localization
+        /// is not available.
+        /// </param>
+        /// <param name="ifNoneMatch">
+        /// Passing the value of the ETag response header in an If-None-Match field will allow the service to optimize the response.
+        /// If the resource has not been modified, the service will return status code 304 and an empty response body.
+        /// </param>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The response returned from the service. Details of the response body schema are in the Remarks section below. </returns>
+        internal virtual async Task<Response> GetLanguagesAsync(string clientTraceId = null, string scope = null, string acceptLanguage = null, ETag? ifNoneMatch = null, RequestContext context = null)
```

Comment:
@MikeyMCZ Do you want to make the protocol method internal here? If so, you should mark the operation with decorator `@protocol(false)` like this https://github.com/Azure/azure-rest-api-specs/pull/23681

Reply:
cc @lmazuel  @m-nash 

Instead of adding the customized operation, seems it is better to update the typespec to add the decorator.